elements/drag-drop-elements/pl-drag-drop-element.css

```diff
@@ -0,0 +1,79 @@
+.connectedSortable {
+  /* applies to MCQ options draggable blocks*/
+  border: 3px solid #eee;
+  border-radius: 4px;
+  /*width: 302px;*/
+  background-color: #efefff;
+  min-width: 200px;
+  width: 350px;
+  min-height: 20px;
+  list-style-type: none;
+  margin: 0;
+  padding: 5px 0 20px 0;
+  float: left;
+  margin-right: 50px;
+}
+/* Dropzones are the answer boxes, which should be denoted
+   with an extra yellow background sheen */
+.dropzone {
+  background-color: #FFA;
+  min-height: 30px;
+}
+
+.info {
+  /*style for question instruction blocks*/
+  background-color: transparent !important;
+  border: none !important;
+}
+
+.code {
+  /*style for question instructions that are programming code*/
+  font-family: "Lucida Console", Monaco, monospace;
+}
+
+
```

Comment:
```suggestion
```
I don't think this `.code` class is being used anywhere? If it is and I missed it, can we just use native bootstrap `.text-monospace` instead?

Reply:
yeah, this is a relic from the past, where we used `pl-info` blocks instead of `leading-code`. This could be safely removed